src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,36 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
```

Comment:
Is `exfmt` appropriate now? Maybe a better name is `Elixir Format" since it is an official formatter.

Reply:
:+1: